Admin/AbstractAdmin.php

```diff
@@ -536,6 +547,13 @@
     private $breadcrumbsBuilder;
 
     /**
+     * The formBuilder component.
+     *
+     * @var AdminBuilderInterface
+     */
+    private $adminBuilder;
```

Comment:
If you remove the getter and the setter for this property on next major, what would need it for? What did I miss?